app/models/king.rb

```diff
@@ -1,8 +1,12 @@
 class King < Piece
-  # ensures king piece can only move 1 square in any direction
+
   def valid_move?(x,y)
-    # checks critera for valid_move? that apply to all pieces and returns false if fails
-    return false unless super(x,y)
+    # checks critera for valid_move? that apply to all pieces and returns false if fails unless can_castle
+    (normal_valid_move?(x,y) && super(x,y)) || can_castle?(x,y)
+  end
+
+  # ensures king piece can only move 1 square in any direction
```

Comment:
Ooops! I clicked on the little accordion button on the side left navigation and got to see that that bit of code wasn't taken out.  Sorry about that!